config/jobs/cert-manager/cert-manager/cert-manager-presubmits.yaml

```diff
@@ -342,20 +337,15 @@ presubmits:
           options:
             - name: ndots
               value: "1"
-    # An E2E test job to allow us to manually trigger the Venafi  TPP E2E tests
-    # with the following GitHub comment:
-    #
-    #  /test pull-cert-manager-issuers-venafi-tpp
-    #
-    # See https://github.com/cert-manager/cert-manager/issues/3555
-    #
     - agent: kubernetes
       always_run: false
       annotations:
         description: Runs the E2E tests with 'Venafi TPP' in name
+        testgrid-alert-email: cert-manager-dev-alerts@googlegroups.com
         testgrid-create-test-group: "true"
         testgrid-dashboards: jetstack-cert-manager-presubmits-blocking
-      branches: []
+      branches:
+        - master
```

Comment:
note: I don't think the fact that `branches` was empty here mattered, because this was an optional presubmit. Still, I think we should set it now.